src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -261,30 +223,27 @@ function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, pol
 
     /**
      * Retrieve verified business bank account currently being set up.
-     * @param {boolean} ignoreLocalCurrentStep Pass true if you want the last "updated" view (from db), not the last "viewed" view (from onyx).
+     * @param ignoreLocalCurrentStep Pass true if you want the last "updated" view (from db), not the last "viewed" view (from onyx).
      */
-    function fetchData(ignoreLocalCurrentStep) {
+    function fetchData(ignoreLocalCurrentStep?: boolean) {
         // Show loader right away, as optimisticData might be set only later in case multiple calls are in the queue
         BankAccounts.setReimbursementAccountLoading(true);
 
         // We can specify a step to navigate to by using route params when the component mounts.
         // We want to use the same stepToOpen variable when the network state changes because we can be redirected to a different step when the account refreshes.
         const stepToOpen = getStepToOpenFromRouteParams(route);
-        const subStep = achData.subStep || '';
-        const localCurrentStep = achData.currentStep || '';
+        const subStep = achData?.subStep ?? '';
+        const localCurrentStep = achData?.currentStep ?? '';
         BankAccounts.openReimbursementAccountPage(stepToOpen, subStep, ignoreLocalCurrentStep ? '' : localCurrentStep, policyID);
     }
 
-    useEffect(
-        () => {
-            fetchData();
-            return () => {
-                BankAccounts.clearReimbursementAccount();
-            };
-        },
+    useEffect(() => {
+        fetchData();
+        return () => {
+            BankAccounts.clearReimbursementAccount();
+        };
         // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
```suggestion
        // eslint-disable-next-line react-hooks/exhaustive-deps The empty dependency array ensures this runs only once after the component mounts.
```

Move the comment to here

Reply:
Not sure why but when doing that there is an eslint error
<img width="1269" alt="image" src="https://github.com/Expensify/App/assets/25230417/b6e217af-1aae-4750-abc0-a302ea3fb75d">